source/containers.tex

```diff
@@ -10515,7 +10534,10 @@
     class span;
 
   template<class ElementType, size_t Extent>
-    inline constexpr bool ranges::enable_safe_range<span<ElementType, Extent>> = true;
+    inline constexpr bool ranges::enable_view<span<ElementType, Extent>> =
+      Extent == 0 || Extent == dynamic_extent;
```

Comment:
`span::span()` has the operands to `||` in reverse order.

Reply:
This issue (LWG3326) was inherited from motions-2020-02-lwg-2.  That said, I don't see what your trying to point out.  Anyway, please move this to lwg-2.